front-end-components/src/index.css

```diff
@@ -210,6 +210,10 @@
   color: inherit;
 }
 
+.composed-container {
+  min-height: 200px;
```

Comment:
Unfortunately it looks like this client rendered component also includes the three stat boxes as well as the chart itself, so this will need to be increased slightly, to something like `335px`, which should also cover the responsiveness.

![image](https://github.com/user-attachments/assets/cee50f8c-6163-4de8-921a-24891a88474c)

Reply:
updated in the latest commit